tests/auxtel/test_latiss_wep_align.py

```diff
@@ -0,0 +1,676 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import random
+import unittest
+import asyncio
+import numpy as np
+import warnings
+import os
+
+import pytest
+
+try:
+    # TODO: (DM-24904) Remove this try/except clause when WEP is adopted
+    from lsst.ts import wep
+
+    WEP_AVAILABLE = True
+except ImportError:
+    WEP_AVAILABLE = False
+    warnings.warn("Could not import cwfs package. Most tests will be skipped.")
+
+from lsst.ts import salobj
+from lsst.ts import standardscripts
+from lsst.ts import externalscripts
+from lsst.ts.externalscripts.auxtel import LatissWEPAlign
+import lsst.daf.butler as dafButler
+from lsst.utils import getPackageDir
+import logging
+
+# Make matplotlib less chatty
+logging.getLogger("matplotlib").setLevel(logging.WARNING)
+# Make obslsst translators less chatty
+logging.getLogger("lsst.obs.lsst.translators").setLevel(logging.WARNING)
+logging.getLogger("astro_metadata_translator").setLevel(logging.WARNING)
+logging.getLogger("flake8.style_guide").setLevel(logging.WARNING)
+
+
+logger = logging.getLogger(__name__)
+logger.propagate = True
+
+random.seed(47)  # for set_random_lsst_dds_partition_prefix
+# Check to see if the test data is accessible for testing at NCSA
+# Depending upon how we load test data this will change
+# for now just checking if butler is instantiated with
+# the default path used on the summit and on the NTS.
+
+# DATAPATH = "/readonly/repo/main" # NTS - obsolete
+DATAPATH = "/repo/LATISS"  # SUMMIT (warning!)
+# DATAPATH = "/repo/LATISS" # TTS (TBC)

```

Comment:
you ran these tests on the TTS right? I think the datapath is the same so you can remove line 67 and make this SUMMIT and TTS

Reply:
Yes, I ran them on TTS.. All these came from the original file.. I'll cleanup both..